src/code/z_nulltask.c

```diff
@@ -0,0 +1,27 @@
+#include "global.h"
+
+/**
+ * Blocks the current thread until all queued scheduler tasks have completed.
```

Comment:
```suggestion
 * Yields until all queued scheduler tasks have completed.
```
I learned new words and I'm not afraid to use them!
is that indeed yielding though?

Reply:
It would yield to other threads, but I think that's more an implementation detail and would prefer to describe what's happening with the current thread specifically since that's what's most important here